<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_ConfigModel</class>
 <widget class="QDialog" name="Dialog_ConfigModel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Config Invoice</string>
  </property>
  <widget class="QPushButton" name="pushButton_Close">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>560</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>211</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Step 1: Choose one customer of invoice</string>
   </property>
   <widget class="QPushButton" name="pushButton_CreateInvoice">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1.Create Invoice</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Customer">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>40</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Customer (*)</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableView" name="tableView_InvoiceHeader">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>761</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Clear">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>560</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>270</y>
     <width>241</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Step 2: Add items to invoice</string>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantity (*)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Item (*)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Quantity">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Item">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Money</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Money">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AddItem">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>2.Add Item</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>270</y>
     <width>291</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Step 3: Preview and submit invoice</string>
   </property>
   <widget class="QTableView" name="tableView_InvoiceDetail">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>271</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SaveInvoice">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>3. Save Invoice</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_InvoiceCode">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Invoice Code</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tableView_InvoiceHeader</tabstop>
  <tabstop>pushButton_Clear</tabstop>
  <tabstop>pushButton_Close</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Close</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_ConfigModel</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>710</x>
     <y>560</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
